emailing-service/emailing-service-test/obj/Debug/net8.0/emailing-service-test.AssemblyInfoInputs.cache

```diff
@@ -1 +1 @@
-117ce3bdaf410224f4eeb7d8de86945269cb59bf6ee47225b25eeaeca7921b1f
+3b3cf1d3cf0d930a36fe4c8ebd9b716bf14b87b274eb10728d05ea922fee1b2d
```

Comment:
I am unsure of what this is, can you enlighten me ? :)

Reply:
This is because some other people built the solution on their laptop and when I built it on mine again it just changed the build folder. In short, this is automatic and I didn't write this

Reply:
I see!